src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -42,13 +42,13 @@ public void testSaveAndLoadUsers() throws PersistentDataStoreException {
     String nameOne = "test_username_one";
     String passwordHashOne = "$2a$10$BNte6sC.qoL4AVjO3Rk8ouY6uFaMnsW8B9NjtHWaDNe8GlQRPRT1S";
     Instant creationOne = Instant.ofEpochMilli(1000);
-    User inputUserOne = new User(idOne, nameOne, passwordHashOne, creationOne);
+    User inputUserOne = new User(idOne, nameOne, passwordHashOne, creationOne,"");
 
     UUID idTwo = UUID.fromString("10000001-2222-3333-4444-555555555555");
     String nameTwo = "test_username_two";
     String passwordHashTwo = "$2a$10$ttaMOMMGLKxBBuTN06VPvu.jVKif.IczxZcXfLcqEcFi1lq.sLb6i";
     Instant creationTwo = Instant.ofEpochMilli(2000);
-    User inputUserTwo = new User(idTwo, nameTwo, passwordHashTwo, creationTwo);
+    User inputUserTwo = new User(idTwo, nameTwo, passwordHashTwo, creationTwo,"");
```

Comment:
nitpick: for this and all of the ones above/below that are similar, insert whitespace between the comma and the ""